src/main/java/com/jcabi/aspects/Cacheable.java

```diff
@@ -104,6 +104,12 @@
     boolean forever() default false;
 
     /**
+     * Returns the current store after the expiration, and
+     * then asynchronously update the data.
+     */
+    boolean asyncUpdate() default false;
```

Comment:
@pinaf The user can decide whether to use  asynchronous method through this property. If asyncUpdate equal to true, using the asynchronous mode, or do not use.